Common/UnitDefinitions/BrakeSpecificFuelConsumption.json

```diff
@@ -4,6 +4,7 @@
   "XmlDocSummary": "Brake specific fuel consumption (BSFC) is a measure of the fuel efficiency of any prime mover that burns fuel and produces rotational, or shaft, power. It is typically used for comparing the efficiency of internal combustion engines with a shaft output.",
   "BaseDimensions": {
     "L": -2,
+    "M": 0,
     "T": 2
```

Comment:
I know, I've added it in a couple of places, where there was a hidden dimensions (I thought it could serve as a _tip_) - but I did also toyed with the idea of treating `null` and `0` differently..
I can remove them if you want (once you've reviewed everything)..

Reply:
I think it just adds confusion, at least for me I don't know it's there.

Reply:
Fixed in https://github.com/angularsen/UnitsNet/pull/1473/commits/c326bbf51378dd2cb20c1b0417fab926e09d5a99